backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -31,7 +31,6 @@
 
   @Autowired private OrganizationService _organizationService;
   @Autowired private PersonService _personService;
-  @Autowired private HibernateQueryInterceptor hibernateQueryInterceptor;
```

Comment:
No longer needed because the is `_hibernateQueryInterceptor.startQueryCount` moved down into BaseServiceTest.java.